src/explore-education-statistics-admin/src/pages/bau/BauImportsPage.tsx

```diff
@@ -0,0 +1,94 @@
+import Link from '@admin/components/Link';
+import Page from '@admin/components/Page';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import React from 'react';
+import importStatusBauService from '@admin/services/importStatusBauService';
+
+const BauImportsPage = () => {
+  const { value, isLoading } = useAsyncRetry(() =>
+    importStatusBauService.getAllIncompleteImports(),
+  );
+
+  return (
+    <Page
+      wide
+      breadcrumbs={[
+        { name: 'Platform administration', link: '/administration' },
+        { name: 'Incomplete imports' },
+      ]}
+    >
+      <h1 className="govuk-heading-xl">
+        <span className="govuk-caption-xl">Manage incomplete imports</span>
+        Incomplete imports
+      </h1>
+      <table className="govuk-table">
+        <thead className="govuk-table__head">
+          <tr className="govuk-table__row">
+            {/* EES-1655 */}
+            {/* <th scope="col" className="govuk-table__header">
+              Subject title
+            </th> */}
+            <th scope="col" className="govuk-table__header">
+              Subject ID
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Data file name
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Meta file name
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Data file rows
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Status
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Stage Percentage Complete
+            </th>
+            <th scope="col" className="govuk-table__header">
+              Go to release
+            </th>
+          </tr>
+        </thead>
+        <LoadingSpinner loading={isLoading} text="Loading imports">
```

Comment:
Wrap the entire table with `LoadingSpinner`. It's invalid HTML to render a `div` (which is what `LoadingSpinner` is) as a child of `table`. The console also warns you that this is a problem.

I think this is also a problem for the `BauUsersPage` you've copied this from